uber/models.py

```diff
@@ -1025,6 +1035,7 @@ class Attendee(MagModel, TakesPaymentMixin):
     placeholder   = Column(Boolean, default=False, admin_only=True)
     first_name    = Column(UnicodeText)
     last_name     = Column(UnicodeText)
+    legal_name_   = Column('legal_name', UnicodeText, nullable=True, default=None)
```

Comment:
That makes sense with the non-nullable and just using empty string for it.

Very relatedly, the renaming of the column is a strange thing, since I wasn't really sure what to name it. There's three different behaviors of data here that we want:

* Get legal name if it's set, if not, get full name.
* Get legal name if it's set, if not, empty string.
* Get/set legal name in raw form, having None as a value.

Those three need three different names, which is why there's the odd underscore there, because I'm guessing we want "attendee.legal_name" to be the first form, and the field is the third form, but I also didn't feel like having a column in the DB with an underscore would be good, hence the rename.

However, if we require non-null and use the empty string to mark absense, then it collapses to two forms. In that case, I would say that `attendee.legal_name` should be the first and `attendee.legal_name_opt` should be the second, but also having the no-rename requirement makes the column be named `legal_name_opt` in the DB. Is that okay?